### DISCUSSION TITLE
An option to turn off "Exit confirmation"

### DISCUSSION THREAD

**Discussion by rvnexe** [ppy/osu#22846] [Ideas] [answered]
When you do Alt + F4 to quit the game, sometimes that confirmation dialog can annoy some users (maybe me too though)
So, there should be a option in settings to disable this dialog and directly close the game when you press Alt + F4

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/3952

**Reply by rvnexe** [reply]
My idea is different. let I explain it with a photo:
![image](https://user-images.githubusercontent.com/89677246/224667309-d4aeaf59-f333-4bd5-a3e1-bf9f0aad73ee.png)
Something like this maybe?
(Made it with photoshop)

**Reply by rvnexe** [reply]
![image](https://user-images.githubusercontent.com/89677246/224667815-2432a245-c555-473a-9836-21636d7182ce.png)
or maybe like this

**Reply by peppy** [reply]
It's not going to happen, pleas read the discussion elsewhere to understand why (there are cases we DO want to confirm).

**Reply by khogeland** [reply]
@peppy where is said discussion? I find this behavior annoying when I've left the game open in the background and can't quickly close it from the taskbar without tabbing back in

**Reply by Joehuu** [reply]
There's a pending change for this: https://github.com/ppy/osu/pull/26558.
